src/app/store-stats/ClarityCharacterStat.m.scss

```
@@ -0,0 +1,68 @@
+@use '../variables' as *;
+@use '../dim-ui/tooltip-mixins' as *;
+
+.communityInsightSection {
+  @include tooltip-section-color($communityBlue);
+  h3 {
+    opacity: 0.6;
+    margin: 0;
+    font-size: inherit;
+  }
+
+  table {
+    border-collapse: collapse;
+    width: 100%;
+  }
+
+  th {
+    font-weight: normal;
+  }
+
+  th,
+  td {
+    opacity: 0.5;
+    &:nth-child(2n) {
+      text-align: right;
+      padding-left: 6px;
+      font-variant-numeric: tabular-nums;
+    }
+  }
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  tbody th {
+    text-align: right;
+    padding-right: 6px;
+    opacity: 1;
+
+    > * {
+      white-space: nowrap;
+    }
+  }
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  thead th {
+    border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+    &:first-child {
+      border: none;
+    }
+  }
+
+  img {
+    vertical-align: bottom;
+    margin-right: 4px;
+  }
+
+  .currentColumn {
+    opacity: 1;
+  }
+}
+
+.unit {
+  font-weight: normal !important;
+  font-size: 0.85em;
```

```suggestion
  font-size: 0.85em;
  line-height: 1em;
```
This should keep the units on the same baseline as the value.